### DISCUSSION TITLE
China pool mining.

### DISCUSSION THREAD

**Discussion by KIMB0212** [Chia-Network/chia-blockchain#5340] [General]
It is not possible that in China they have already created the pool and are only undermining them. It is not fair to all the people who believe in the project and have invested and are holding a handful of flies.

**Comment by room101-dev** [comment]
Join the CHinese pool? What is stopping you?

The more important rhetorical question is "IS the entire project", really a chinese operation?? Don't joke, think about bitcoin-gold, it was funded by China, and the dev's were in Ukraine, they down-played the Chinese control

Think about this too, we can't even see the source for the plot-creation in chia, because its top secret, that is VERY chinese not making source public, CHIA likes to say 'the source is on github', NO ITS NOT, not the source that counts.

I'm burning up a $200 NVME every two weeks running this plot-create software, and if I had the source I would fix the problem, just like I do in GPU mining. I don't burn up my GPU cards, and I hate to see my NVME's dying when they're only at 25% of TBW ( warranty ) ( samsung 980 pro 's )

Lot's of mis-information here that you 'cannot join the pool', hell as far as I can see it was available months ago 'hpool', and its in english. So you can have 100's, if not 1,000's of plots and never win one chia, but you could have been 'mining' on the pool all along. Go figure.

**Reply by climba03003** [reply]
> Think about this too, we can't even see the source for the plot-creation in chia, because its top secret

Check this, and it is what you need for `plot-creation in chia`.
https://github.com/Chia-Network/chiapos
